src/components/forms/PlanForm/index.tsx

```diff
@@ -1261,7 +1278,7 @@ const PlanForm = (props: PlanFormProps) => {
   );
 };
 
-export const defaultProps: PlanFormProps = {
+export const defaultProps: Omit<PlanFormProps, 'actionCreator'> = {
```

Comment:
why is it necessary to change this type?

Reply:
updated.